scripts/eq_translations_check.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import requests
+import sys
+
+import eq_translations
+
+from requests.exceptions import ConnectionError
+
+logger = logging.getLogger(__name__)
+version = f"v{eq_translations.__version__}"
+
+try:
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-translations/releases"
+    )
+    if response.status_code == 200:
+        latest_tag = response.json()[0]["tag_name"]
+        if latest_tag != version:
+            logger.error(
+                'Newer version of eq-translations available, use "pipenv update"'
```

Comment:
I guess this sound say, what the latest version is, as `pipenv update` won't bump the version in the pipfile.

Reply:
It does say now what the latest version is, should I remove that `pipenv update` part?

Reply:
Maybe something like `eq-translations is out of date. Update to {version}.`?

Reply:
Ok, sorted! 👍